SearchInput.js

```diff
@@ -0,0 +1,105 @@
+'use strict';
+
+import React, { Component } from 'react';
+import { StyleSheet, Text, View, Image, TouchableOpacity } from 'react-native';
+import { GooglePlacesAutocomplete } from 'react-native-google-places-autocomplete';
+import { apiKey_google_place, apiKey_openweather } from './apiKey';
+
+class SearchInput extends Component {
+	constructor(props) {
+	  super(props);
```

Comment:
you can define state as class properties. If you do that way you don't need to create a class contructor.
https://michalzalecki.com/react-components-and-class-properties/